ksql-engine/src/test/java/io/confluent/ksql/testutils/KafkaEmbedded.java

```diff
@@ -60,52 +60,35 @@
   /**
    * Creates and starts an embedded Kafka broker.
    *
-   * @param config Broker configuration settings.  Used to modify, for example, on which port the
-   *               broker should listen to.  Note that you cannot change some settings such as
-   *               `log.dirs`, `port`.
+   * @param config Broker configuration settings.  Used to modify, for example, on what listeners
```

Comment:
nit: `...for example, the listeners the broker should use."

Reply:
done